### DISCUSSION TITLE
Option to disable the formatting editor preview

### DISCUSSION THREAD

**Discussion by AndreaMonzini** [kmwallio/ThiefMD#157] [Ideas]
Hello,
thank you very much for the project and for the .fountain support !

It would be be useful to have an option to disable the formatting editor preview for .md and .fountain projects.
So i can write in plain text without the formatting preview in one window and the full preview in the external window ( side by side).

The formatting editor is very nice but sometimes i prefer to work with the "raw" plain text.

Thank you !

**Comment by kmwallio** [comment]
With disabling formatting, is that syntax highlighting as well? Or just the auto alignment?

**Comment by AndreaMonzini** [comment]
Personally i would prefer the option to disable both syntax highlighting and auto alignment.
As alternative you could add two separate options:
1- disable syntax highlighting
2- disable auto alignment

If useful i can test the beta :)

Thank you !
